unit2-java/lesson18.rst

```diff
@@ -17,3 +20,4 @@ Objectives
 - Explain how content negotiation works, and which HTTP headers are necessary for this
 - Explain idempotence in REST
```

Comment:
Is "idempotence" really a word?  If so, *italics* or **bold** to emphasize that it is NOT a weird typo.